src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const smallTerm = 3;
+  const smallTermDiscount = 20;
+  const bigTerm = 7;
+  const bigTermDiscount = 50;
+  const numberOfDays = days;
+  const basePrice = days * 40;
+
+  if (numberOfDays < smallTerm) {
+    return basePrice;
+  }
+
+  if (numberOfDays < bigTerm) {
+    return basePrice - smallTermDiscount;
```

Comment:
The condition `numberOfDays < bigTerm` should be `numberOfDays >= smallTerm && numberOfDays < bigTerm` to ensure that the small term discount is only applied when the number of days is between 3 and 6 inclusive. This ensures that the discount logic aligns with the intended discount structure.